LayoutTests/fast/css/link-style-color-invalidation.html

```diff
@@ -0,0 +1,16 @@
+<style>
+    * {
+        margin:0;
+        padding:0;
+    }
+    a:link { color: green; }
+    a:hover { color: green; }
+</style>
+
+<a href="">Shoud be green when hovered</a>
```

Comment:
"Shoud"

Reply:
Why not make the `<a>` much bigger so you don't need the `margin:0; padding:0` stuff?

Reply:
Don't even need the padding/margin altogether. Removed ✔️